TeamSnapSDK/SDK/DataLayer/TSDKObjectsRequest.h

```diff
@@ -28,4 +28,6 @@
 + (NSArray *_Nonnull)knownCompletionTypes;
 + (void)dumpCompletionTypes;
 
++ (void)listEventsForTeams:(NSArray * _Nonnull)teams pageNumber:(NSNumber *_Nullable)pageNumber pageSize:(NSNumber *_Nullable)pageSize startDate:(NSDate*_Nullable)startDate endDate:(NSDate*_Nullable)endDate completion:(TSDKArrayCompletionBlock _Nullable)completion;
```

Comment:
- [ ] This should have an object type for the items in the teams Array.  (is it looking for IDs? TSDKTeams?)
(if it's looking for TeamIds I might even rename the method to listEventsForTeamIds
- [ ] The return type should probably be TSDKEventArrayCompletionBlock